src/header.js

```diff
@@ -60,6 +63,15 @@ class Header extends EthObject {
       if (rpcResult.withdrawalsRoot !== undefined && rpcResult.withdrawalsRoot !== null) {
         data.push(toBuffer(rpcResult.withdrawalsRoot));
       }
+      if (rpcResult.blobGasUsed !== undefined && rpcResult.blobGasUsed !== null) {
+        data.push(toBuffer(rpcResult.blobGasUsed));
+      }
+      if (rpcResult.excessBlobGas !== undefined && rpcResult.excessBlobGas !== null) {
+        data.push(toBuffer(rpcResult.excessBlobGas));
+      }
+      if (rpcResult.parentBeaconBlockRoot !== undefined && rpcResult.parentBeaconBlockRoot !== null) {
+        data.push(toBuffer(rpcResult.parentBeaconBlockRoot));
+      }
```

Comment:
Should we fix function `fromWeb3` for `blobGasUsed` and `excessBlobGas`?

Reply:
70f6cdf3277a291d6c27e2df23217b79d999a8be